### PR TITLE
✨ fix LDAP login failed

### PR DESCRIPTION
**Pull request by tonycody**
✨ fix LDAP AD login Ignore partial result exception

#9862 

Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/LdapAuthConfig.java [hunk 1]
```diff
@@ -40,47 +40,55 @@
 @Configuration(proxyBeanMethods = false)
 @EnableAutoConfiguration(exclude = LdapAutoConfiguration.class)
 public class LdapAuthConfig {
-    
+
```
Comment (KomachiSion) [inline plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/LdapAuthConfig.java:43]: Please use nacos code style to reformat codes

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/LdapAuthConfig.java (hunk 1 above)
Comment (tonycody) [reply plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/LdapAuthConfig.java:43]: 构建过的，风格校验没问题。

Comment (tonycody) [reply plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/LdapAuthConfig.java:43]: 关于风格的问题，咱是不是可以通过：spotless 或 Spring Java Format 直接规整下？而不是检查。

Comment (KomachiSion) [reply plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/LdapAuthConfig.java:43]: 自动规整的问题可以研究下， 不过这个PR肯定没有用nacos code style进行reformat， 因为缩进被去掉了。

**Review by KomachiSion — CHANGES_REQUESTED** [review]
缩进问题依旧没有修改

**Review by KomachiSion — APPROVED** [review]

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**